MiniAn/minian_main.py

```diff
@@ -467,6 +471,115 @@ def load_doric_to_xarray(
 
     return varr, file_
 
+            
+def cross_register(AC, A, minian_parameters):
+    perform_cross_reg = minian_parameters.params_cross_reg["cross_reg"]
+    if not perform_cross_reg:
+        return A
+
+    print(mn_defs.Messages.CROSS_REGISTRATING , flush=True)
+    param_dist = 5 # Defines the max dist between centroids on different sessions to consider them as same cell
```

Comment:
It has to be part of the cross registration parameters. We mind need to predefine it with the neuron diameter